javascript/javascript1/week3/homework-wk 3/homework-wk3.js

```diff
@@ -0,0 +1,222 @@
+"use strict"
+
+
+// Item Array Removal
+
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = ('Ahmad');
+//names.splice(1,1);
+
+for ( let i = 0 ; i < names.length ; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i ,1) ;
+  }
+}
+console.log(names); 
+
+
+
+//When will we be there
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function convertTravelTime(travelInformation){
+   const timeInHours =  Math.floor(travelInformation.destinationDistance / travelInformation.speed);
+   const timeInMinutes = Math.floor((((travelInformation.destinationDistance / travelInformation.speed)-timeInHours)*60));
+   return `${timeInHours} hours and ${timeInMinutes} minutes`;
+
+  }
+
+const travelTime = convertTravelTime(travelInformation);
+console.log(travelTime)
+
+
+
+//Series duration of my life
+
+const seriesDurations = [
+    {
+      title: 'Game of thrones',
+      days: 3,
+      hours: 1,
+      minutes: 0,  
+    },
+    {
+      title: 'Sopranos',
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: 'Vikings',
+      days: 2,
+      hours: 9,
+      minutes: 55,
+    }
+]
+
+
+
+const lifeSpanInhours = 80* 365 * 24;
+
+function sumTotalOfHoursSpent(object){
```

Comment:
the name `object` is too generic. What does this object represent?

Reply:
it's to represent the objects in the series duration array. I changed the name of the parameter.